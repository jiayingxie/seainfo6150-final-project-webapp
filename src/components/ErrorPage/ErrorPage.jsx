import React from 'react';
import { Link, Route } from "react-router-dom";
import Home from '../Home/Home.jsx';
import styles from './ErrorPage.module.css';

const ErrorPage = () => {
    return (
        <div className={styles.container}>
            <h1>Sorry, could not find the page.</h1>
            <h2>Go back to the <Link to="/">HOME</Link> page.</h2>
            <Route path="/" exact component={Home} />
        </div>
    )
}

export default ErrorPage;