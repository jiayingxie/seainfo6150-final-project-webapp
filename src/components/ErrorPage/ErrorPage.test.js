import React from "react";
import ErrorPage from "./ErrorPage";

describe("ErrorPage tests", () => {
    it("renders correctly", () => {
        const { container } = render(<ErrorPage />);
        expect(container).toMatchSnapshot();
    });
});
