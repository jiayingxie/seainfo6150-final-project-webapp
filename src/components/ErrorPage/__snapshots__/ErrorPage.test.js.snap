// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorPage tests renders correctly 1`] = `
<div>
  <div
    class="container"
  >
    <h1>
      Sorry, could not find the page.
    </h1>
    <h2>
      Go back to the 
      <a
        href="/"
      >
        HOME
      </a>
       page.
    </h2>
    <div
      class="container"
    >
      <h1>
        Welcome to Graduate, College of Engineering
      </h1>
       
      <br />
      <img
        alt="tree frog"
        class="image"
        src="treefrog.jpg"
      />
    </div>
  </div>
</div>
`;
